jb/itch5/array_based_order_book.hpp

```diff
@@ -22,14 +21,16 @@ namespace itch5 {
 
 // forward declaration of
 void validate_add_order_params(int, price4_t px = price4_t(0));
+void handle_exception(std::string, std::size_t, price4_t, int);
+void handle_exception(
+    std::string, std::size_t, std::size_t, std::size_t, price4_t, int);
 
 namespace defaults {
```

Comment:
I think it is possible to move the whole namespace defaults to the .cpp file, right?